lib/prerequisite.js

```diff
@@ -93,18 +94,7 @@ module.exports = (input, pkg, options) => {
 			task: async () => {
 				await execa('git', ['fetch']);
 
-				const getTagVersionPrefix = () => {
-					if (options.yarn) {
-						return execa.stdout('yarn', ['config', 'get', 'version-tag-prefix']);
-					}
-
-					return execa.stdout('npm', ['config', 'get', 'tag-version-prefix']);
-				};
-
-				let tagPrefix = 'v';
-				try {
-					tagPrefix = await getTagVersionPrefix();
-				} catch (_) {}
+				const tagPrefix = await getTagVersionPrefix();
```

Comment:
You are not passing the `yarn` option. Same with the other call to `getTagVersionPrefix`.